x-pack/plugins/security_solution/common/types/timeline/store.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ColumnHeaderOptions,
+  ColumnId,
+  RowRendererId,
+  TimelineExpandedDetail,
+  TimelineTypeLiteral,
+} from '.';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { Filter } from '../../../../../../src/plugins/data/public';
+
+import { Direction } from '../../search_strategy';
+import { DataProvider } from './data_provider';
+
+export type KueryFilterQueryKind = 'kuery' | 'lucene' | 'eql';
```

Comment:
consider moving this to the `kbn-securitysolution-t-grid` package, because it overlaps with `x-pack/plugins/timelines/common/types/timeline/store.ts`

Reply:
@kqualters-elastic will take care of it! ❤️